grails-app/controllers/org/pih/warehouse/core/OrganizationController.groovy

```diff
@@ -12,33 +12,45 @@ package org.pih.warehouse.core
 import org.pih.warehouse.inventory.InventoryItem
 import org.pih.warehouse.product.Product
 import org.springframework.transaction.TransactionStatus
+import org.grails.plugins.csv.CSVWriter
 
 class OrganizationController {
 
     def identifierService
+    def organizationService
     def scaffold = true
 
     def search = {
+        def organizationInstanceList = organizationService.getOrganizations(params)
+        render(view: "list", model: [organizationInstanceList:organizationInstanceList, organizationInstanceTotal:organizationInstanceList.totalCount])
+        if (params.format && organizationInstanceList) {
+            redirect(controller: "organization", action: "download", params: params)
```

Comment:
I wanted that to be called separately, not through the search action. Now you have to get organization list two times.

Reply:
@awalkowiak good point, done!